### PR TITLE
[docs]<docs>Add file system benchmark tools docs

### PR DESCRIPTION
**Pull request by hubgeter**
## Proposed changes
Add file system benchmark tools docs
Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/lakehouse/fs_benchmark_tool.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+---
+{
+    "title": File system benchmark tools",
```
Comment (morningman) [inline docs/en/docs/lakehouse/fs_benchmark_tool.md:3]:
```suggestion
    "title": "File system benchmark tools",
```

**Comment by morningman** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by kaka11chen — APPROVED** [review]

**Comment by kaka11chen** [comment]
LGTM

**Pull request merged**